advisories/c20230118.md

```diff
@@ -29,7 +29,7 @@ All users assigned the Developer role in a CockroachDB Cloud organization will n
 - `CONTROLJOB`
 - Cluster node metadata
 
-See [Role Options](../{{site.current_cloud_version}}/alter-user.html#role-options) for more information on these roles.
+See [Role Options](../{{site.current_cloud_version}}/alter-user.html#{% if site.current_cloud_version == "v22.1" %}parameters{% else %}role-options{% endif %}) for more information on these roles.
```

Comment:
#role-options also present in [v22.2](https://www.cockroachlabs.com/docs/stable/alter-user.html#role-options) though (and all future).

Reply:
Yep, but the issue is `current_cloud_version` is listed as v22.1, where the heading doesn't exist. I can remove this block and replace it with `#role-options` once that changes.

Reply:
Sounds good, but it'd be another thing to remember that's on you if you want to shoulder it. I don't suppose I see another way though, you're right.